display_list/benchmarking/dl_complexity.cc

```diff
@@ -24,7 +24,11 @@ DisplayListComplexityCalculator* DisplayListComplexityCalculator::GetForBackend(
     GrBackendApi backend) {
   switch (backend) {
     case GrBackendApi::kMetal:
+#if !SLIMPELLER
```

Comment:
I've also resorted to putting giant if !SLIMPELLER guards around the header and the implementation for good measure. That way, unknown uses are shaken out and the stuff isn't even compiled in.

Reply:
Done